OneCommunityGlobal/HGNRest
# Vishala Types json file fix

# Description
This PR is to file the build of json file. 

## Related PRS (if any):
To test this backend PR you need to checkout the development frontend branch.
…

## Main changes explained:
- Moved JSON file outside src folder

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. log as admin user
5. Go to http://localhost:3000/bmdashboard/materials/add
6. Test the Add material page
